<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>907</width>
    <height>714</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>0</y>
      <width>731</width>
      <height>201</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Ubuntu</family>
      <pointsize>20</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(51, 209, 122);
background-color: rgb(255, 255, 255);
font: 20pt &quot;Ubuntu&quot;;
</string>
    </property>
    <property name="text">
     <string>         Chào mừng bạn đã đến với Nha Khoa IoT</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>0</y>
      <width>191</width>
      <height>201</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../.designer/backup/logo-illustration-of-a-combination-of-teeth-and-toothbrush-very-suitable-for-dental-companies-free-vector.jpg</pixmap>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>200</y>
      <width>921</width>
      <height>521</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(98, 160, 234);</string>
    </property>
    <property name="currentIndex">
     <number>1</number>
    </property>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>Lịch</string>
     </attribute>
     <widget class="QCalendarWidget" name="calendarWidget">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>60</y>
        <width>561</width>
        <height>281</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(53, 132, 228);</string>
      </property>
     </widget>
     <widget class="QListWidget" name="listWidget">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>60</y>
        <width>256</width>
        <height>192</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(53, 132, 228);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="changeState">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>310</y>
        <width>251</width>
        <height>25</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(53, 132, 228);
border-radius:10px;</string>
      </property>
      <property name="text">
       <string>Thay đổi trạng thái </string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>Hóa đơn</string>
     </attribute>
     <widget class="QLabel" name="label_3">
      <property name="geometry">
       <rect>
        <x>320</x>
        <y>0</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Hóa đơn dịch vụ</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_4">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>30</y>
        <width>41</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Ngày:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_5">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>80</y>
        <width>151</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Bệnh nhân:</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_7">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>30</y>
        <width>101</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>ID cuộc hẹn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="cfRc">
      <property name="geometry">
       <rect>
        <x>70</x>
        <y>380</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Xác nhận hóa đơn</string>
      </property>
     </widget>
     <widget class="QPushButton" name="sendRc">
      <property name="geometry">
       <rect>
        <x>630</x>
        <y>380</y>
        <width>171</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Gửi hóa đơn cho Admin</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="apmID">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>30</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="ptName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>70</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_8">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>120</y>
        <width>61</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Dịch vụ:</string>
      </property>
     </widget>
     <widget class="QComboBox" name="service">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>120</y>
        <width>111</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nhổ răng</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Trồng răng</string>
       </property>
      </item>
     </widget>
     <widget class="QSpinBox" name="quantity">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>190</y>
        <width>44</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_9">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>190</y>
        <width>67</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Số lượng</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>50</x>
        <y>130</y>
        <width>91</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Bác sỹ:</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="dtName">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>130</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
     </widget>
     <widget class="QDateEdit" name="ngayKham">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>30</y>
        <width>110</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="Tongtien">
      <property name="geometry">
       <rect>
        <x>200</x>
        <y>230</y>
        <width>401</width>
        <height>71</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QTimeEdit" name="timeEdit">
      <property name="geometry">
       <rect>
        <x>530</x>
        <y>80</y>
        <width>118</width>
        <height>26</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label_6">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>80</y>
        <width>81</width>
        <height>17</height>
       </rect>
      </property>
      <property name="text">
       <string>Giờ khám:</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>Tài khoản</string>
     </attribute>
     <widget class="QPushButton" name="pushButton">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>370</y>
        <width>89</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Đăng xuất</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>380</x>
        <y>320</y>
        <width>151</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string>Cập nhật tài khoản </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_5">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>240</y>
        <width>81</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Ngày </string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_2">
      <property name="geometry">
       <rect>
        <x>420</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Số điện thoại</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_3">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>110</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Mật khẩu</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_4">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>180</y>
        <width>321</width>
        <height>25</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
      <property name="placeholderText">
       <string>Email</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_7">
      <property name="geometry">
       <rect>
        <x>480</x>
        <y>240</y>
        <width>141</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Năm</string>
      </property>
     </widget>
     <widget class="QComboBox" name="comboBox_4">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>40</y>
        <width>86</width>
        <height>25</height>
       </rect>
      </property>
      <item>
       <property name="text">
        <string>Nam</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Nữ</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>Lê Hoàng Anh</string>
       </property>
      </item>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>240</y>
        <width>61</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Tháng</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>113</width>
        <height>25</height>
       </rect>
      </property>
      <property name="placeholderText">
       <string>Tên tài khoản</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>907</width>
     <height>28</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(98, 160, 234);
background-color: rgb(53, 132, 228);
font: 14pt &quot;Ubuntu&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QMenu" name="menuTrang_ch">
    <property name="title">
     <string>Trang chủ</string>
    </property>
   </widget>
   <addaction name="menuTrang_ch"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
